docs/openapi/components/schemas/common/DCSATransportDocument.yml

```diff
@@ -186,5 +186,169 @@ required:
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/242"
+    "id": "http://example.org/credentials/",
+    "type": [
+      "VerifiableCredential"
+    ],
+    "issuanceDate": "2021-02-04T20:29:37+00:00",
+    "issuer": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+    "credentialSubject": {
+      "type": "DCSATransportDocument",
+      "transportDocumentReference": "XMANHR2102045",
+      "shippingInstruction": {
+        "type": "DCSAShippingInstruction",
+        "carrierBookingReference": "XMANHR2102045",
+        "transportDocumentType": "MBL",
+        "shipper": {
+          "type": "Organization",
+          "name": "Xxinau Manufacturing Co. Ltd.",
+          "description": "Advanced Production - Delivered",
+          "address": {
+            "type": "PostalAddress",
+            "streetAddress": "Xin Fei Da Dao 139",
+            "addressLocality": "Xindao",
+            "addressRegion": "Fujian Province",
+            "postalCode": "361100",
+            "addressCountry": "CN"
+          },
+          "email": "xxinau-sales@example.org",
+          "phoneNumber": "+86-555-865-8495"
+        },
+        "consignee": {
+          "type": "Organization",
+          "name": "Better Life Tech",
+          "description": "Better Lives Products",
+          "address": {
+            "type": "PostalAddress",
+            "streetAddress": "Rue de la Poste 272",
+            "addressLocality": "Ramegnies-Chin",
+            "addressRegion": "Hainaut",
+            "postalCode": "7520",
+            "addressCountry": "BE"
+          },
+          "email": "procurement@lifetech-example.org",
+          "phoneNumber": "+32-5555-8495"
+        },
+        "firstNotify": {
+          "type": "Organization",
+          "name": "Better Life Tech",
+          "description": "Better Lives Products",
+          "address": {
+            "type": "PostalAddress",
+            "streetAddress": "Rue de la Poste 272",
+            "addressLocality": "Ramegnies-Chin",
+            "addressRegion": "Hainaut",
+            "postalCode": "7520",
+            "addressCountry": "BE"
+          },
+          "email": "procurement@lifetech-example.org",
+          "phoneNumber": "+32-5555-8495"
+        },
+        "consigneesFreightForwarder": {
+          "type": "Organization",
+          "name": "Intertrans NV [378]",
+          "address": {
+            "type": "PostalAddress",
+            "streetAddress": "Belcrownlaan 25 - 3rd floor",
+            "addressLocality": "Antwerpen",
+            "postalCode": "BE-2100AN",
+            "addressCountry": "BE"
+          },
+          "phoneNumber": "+32-3-201.98.10"
+        },
+        "cargoItems": [
+          {
+            "type": "CargoItem",
+            "cargoLineItems": [
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators",
+                "shippingMarks": "ByAcre Age is the new Black"
```

Comment:
wut is this saying

Reply:
![image](https://user-images.githubusercontent.com/34443212/150126780-67cbdc53-ef6c-483a-b73f-4e81cee19332.png)
"Marks and numbers". It's what's printed on the boxes, assisting consignee/customs/others in determining and organizing what's in the container.